maven-core/src/test/resources/project-dynamism/pom-relative.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<project
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
My comments regarding this broken POMs still stand. No namespace means an invalid POM logically.

Reply:
Is there any reason why they don't use the correct namespace ?

Reply:
ehh.. my IDE try to be more smart then I

Reply:
now - should be ok